fhir-core/src/main/java/com/ibm/fhir/core/FHIRMediaType.java

```diff
@@ -29,6 +35,13 @@
     public final static MediaType APPLICATION_FHIR_NDJSON_TYPE = new MediaType("application", SUBTYPE_FHIR_NDJSON);
 
     public final static String SUBTYPE_FHIR_PARQUET = "fhir+parquet";
-    public static final String APPLICATION_PARQUET = "application/"  + SUBTYPE_FHIR_PARQUET;
+    public static final String APPLICATION_PARQUET = "application/" + SUBTYPE_FHIR_PARQUET;
     public final static MediaType APPLICATION_FHIR_PARQUET_TYPE = new MediaType("application", SUBTYPE_FHIR_PARQUET);
+
+    // Supported values for the MIME-type parameter fhirVersion.
+    // https://www.hl7.org/fhir/http.html#version-parameter
+    // The value of this parameter is the publication and major version number for the specification.
+    public static final String FHIR_VERSION_PARAMETER = "fhirVersion";
+    public static final Set<String> SUPPORTED_FHIR_VERSIONS =
+            Collections.unmodifiableSet(new HashSet<>(Arrays.asList("4.0")));
```

Comment:
I think 4.0.1 is valid too? Right?

Reply:
> Would we want to use 4.0.0 and 4.0.1 as supported versions or is 4.0 sufficient?

Yeah, I wasn't sure about that, based on https://www.hl7.org/fhir/http.html#version-parameter, it says the value of this parameter is the publication and major version number for the specification. So it sounds like it is not supposed to include the minor version.  However, I'm ok with adding 4.0.1 to the list if that makes more sense.

Reply:
Added 4.0.1.